teams/teams-ps/teams/Get-Team.md

```diff
@@ -61,6 +61,13 @@ PS> Get-Team -DisplayName "Sales and Marketing"
 ```
 Returns the team that includes the specified text in its DisplayName. (This acts as a filter rather than an exact match).
 
+### Example 5
+```
+PS> $team=[uri]::EscapeDataString('AB&C')
+PS> Get-Team -DisplayName $team
+```
+Returns the team that includes the specified escaped representation of its DisplayName, useful when the DisplayName has special characters. (This acts as a filter rather than an exact match).
```

Comment:
```suggestion
Returns the team that includes the specified escaped representation of its DisplayName, useful when the DisplayName has special characters. (This acts as a filter rather than an exact match.)
```